aptos-move/framework/aptos-stdlib/sources/data_structures/big_vector.move

```diff
@@ -262,6 +264,19 @@ module aptos_std::big_vector {
         exist
     }
 
+    /// Convert the big vector to a native vector.
+    /// Disclaimer: This function is costly. Use it at your own discretion.
```

Comment:
More details on this would be nice regarding why it's costly. We should also mention this is most useful for view functions, where gas is not charged if called via the fullnode API.